### DISCUSSION TITLE
Firefly Shimmer SMR lost

### DISCUSSION THREAD

**Discussion by Michal1984rls** [iotaledger/firefly#4749] [Help & Support]
Hello. today i made the claim of shimmer but immediately after i had the blackout caused by a thunderstorm. now if I try to recover my SMR it tells me that they are claimed and are in wallet 1, but I created my profile after the blackout and I cannot recover them. how can I do?

**Comment by MiKeF05** [comment]
You have already successfully claimed your SMR token and had a profile in Firefly Shimmer?

**Reply by Michal1984rls** [reply]
I requested the claim but I didn't have a profile

**Reply by Michal1984rls** [reply]
![IMG_20220928_210300](https://user-images.githubusercontent.com/114612749/192867133-2cedcc9f-4082-4ca8-a909-ad38d436dfbe.jpg)

**Reply by MiKeF05** [reply]
If the claiming process was successfull then you can do the following. Create a new profile and choose "Restore Shimmer profile". Then you can restore it with your 24 words of the iota profile with which you staked for SMR. Or if you have a ledger profile then choose the hardware profile.

**Reply by ProdeluxTR** [reply]
I have also a problem. Iclaimed the smr the firefly crashed did it again but when i log it says something went wrong and it goes not further to wallet

**Reply by Michal1984rls** [reply]
Thank you so much my friend.  It worked.  Just restore the wallet with the secret phrases of firefly Iota and not those of firefly shimmer

**Comment by ManniMaker85** [comment]
Ich habe das gleiche Problem, ich habe die SMR beansprucht und wo ich wollte, kam die Fehlermeldung (undefiniert: Konto mit ID 1 kann nicht gefunden werden!) und ich konnte nichts mehr tun, ich habe auch Firefly geschlossen und wollte auch mit neuen ausgehen Erstelltes Profil und behauptete und jetzt sagt es mir, dass die SMR bereits beansprucht wurde und ich nicht abschließen kann und keine SMR auf meiner FireflySMR-Brieftasche habe  <img width="431" alt="Bildschirmfoto 2022-09-30 um 21 13 16" src="https://user-images.githubusercontent.com/114800101/193401853-e31fb435-b213-42e4-9d98-6011af2f03c1.png">

<img width="463" alt="Bildschirmfoto 2022-09-30 um 21 00 09" src="https://user-images.githubusercontent.com/114800101/193401874-704d5308-8d85-4028-8dba-50bdf9b2fbe9.png">

**Comment by 1979aea2022** [comment]
Hola, he cometido un error y me faltan muchos Shimmers en el reclamo se restauró una billetera con un saldo muy pequeño que aparece como reclamado, el problema fue que estaba dando buscar recompensas y el saldo estaba incompleto y le di reclamar recompensa por error  y a la hora de que termina el reclamo como dura tanto deje la computadora sola ella se apagó y cuando volví a restaurar el proceso de reclamo apareció una billetera nada más con un saldo reclamado sumamente pequeño, y las otras dos billeteras no aparecieron siendo que al principio del proceso de búsqueda estaban las tres billeteras, restablecí el perfil de SHIRMER  y aparece una sola billetera con pocos shimmer pera a la hora de darle a buscar dice que hay tres billeteras con el saldo correcto, quisiera saber como puedo reclamar ese saldo de shimmer que no aparece como reclamado.
Muchas gracias.

**Comment by Wolfi2301** [comment]
Ich habe folgendes Problem:
Nach Erstellung (Import mit Backup) eines Profils kam bei der neuen Anmeldung folgendes Bild:

![image](https://user-images.githubusercontent.com/115024205/193860278-12f07eef-6fe3-4bb1-8e9c-47045ecb0594.png)

Fehlerprotokoll:
Error Type: [Preload Context] Error

Error: The profile manager was already initialized.
    at C:\Program Files\Firefly Shimmer\resources\app.asar\public\build\preload.js:2:307346
    at Array.forEach (<anonymous>)
    at EventEmitter.<anonymous> (C:\Program Files\Firefly Shimmer\resources\app.asar\public\build\preload.js:2:307333)
    at EventEmitter.emit (node:events:526:28)
    at Object.onMessage (node:electron/js2c/renderer_init:69:746)


Bitte um Unterstützung.
Wenn ich das Profil neu installiere zeigt es mir an, dass ich bereits die Token beansprucht habe.
Im IOTA Firefly zeigt es mir aber die SMR noch an.

Danke für die Hilfe
